src/devices/video/ef9345.cpp

```diff
@@ -1018,6 +1022,9 @@ void ef9345_device::update_scanline(uint16_t scanline)
 
 	set_busy_flag(104000);
 
+	if (scanline % 10)
+		return;
+
```

Comment:
It’s not a good idea to be doing things this way because the system can “race the beam” and change video registers between scanlines.  Drawing should be clipped to the requested line only.

Reply:
I agree with you.

However, the main change in this PR was about the Quadrichrome palette.

Would it be ok for you if I drop this test and re-submit with just the palette modification ?